Source/Core/Core/DSP/Jit/DSPJitArithmetic.cpp

```diff
@@ -1367,8 +1367,7 @@ void DSPEmitter::lsrn(const UDSPInstruction opc)
 	//		acc <<= -shift;
 	//	}
 
-	CMP(64, R(RDX), Imm8(0));//is this actually worth the branch cost?
-	FixupBranch zero = J_CC(CC_E);
+	FixupBranch zero = J_CC(CC_E);//is this actually worth the branch cost?
```

Comment:
Are you intentionally branching based on the flags generated by SHR?  That isn't a good idea; it causes a pipeline stall in some processors.